operator/src/main/java/oracle/kubernetes/weblogic/domain/model/AuxiliaryImage.java

```diff
@@ -38,17 +37,15 @@
   @EnumClass(ImagePullPolicy.class)
   private String imagePullPolicy;
 
-  @Description("The command for this init container. Defaults to `cp -R $AUXILIARY_IMAGE_PATH/* "
-      + "$AUXILIARY_IMAGE_TARGET_PATH`. This is an advanced setting for customizing the container command for copying "
-      + "files from the container image to the auxiliary image emptyDir volume. Use the `$AUXILIARY_IMAGE_PATH` "
-      + "environment variable to reference the value configured in `spec.auxiliaryImageVolumes.mountPath`, which "
-      + "defaults to \"/auxiliary\". Use '$AUXILIARY_IMAGE_TARGET_PATH' to refer to the temporary directory created by "
-      + "the operator that resolves to the auxiliary image's internal emptyDir volume.")
-  private String command;
+  @Description("The source location of the WebLogic Deploy Tooling installation within the auxiliary image. "
+          + "Defaults to '/auxiliary/weblogic-deploy'. If the value is set to 'None' or no files found at the default "
+          + "location, then the WebLogic Deploy Tooling installation copy is skipped.")
```

Comment:
We are downplaying (but not hiding) the "auxiliary image volume tuning parameters" in 4.0 so I am less enamored with adding the last sentence...

Reply:
The descriptions are traditionally very exact, since this makes the WKO code self documenting (in essence, the descriptions are the best and only "spec" we have).  We can downplay in doc/samples but I think we should not do it here.